index.js

```diff
@@ -17,7 +17,7 @@ module.exports = num => {
 	}
 
 	const exponent = Math.min(Math.floor(Math.log10(num) / 3), UNITS.length - 1);
-	const numStr = Number((num / Math.pow(1000, exponent)).toPrecision(3));
+	const numStr = (num / Math.pow(1000, exponent)).toLocaleString(locale, { minimumFractionDigits: 2, maximumFractionDigits: 2 });
```

Comment:
You can just use argument-less `.toLocaleString()` which will automatically format in the user's locale.

Reply:
Also, keep the `Number()`.